concrete/src/Updater/Migrations/Migrations/Version20170614000000.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20170614000000 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $service = \Core::make('site');
+        $site = $service->getDefault();
+        $siteConfig = $site->getConfigRepository();
+        $url = $siteConfig->get('seo.canonical_ssl_url');
+        if ($url) {
+            $siteConfig->save('seo.canonical_url_alternative', $url);
```

Comment:
Should we see if the `canonical_url` is set here and put the url there if it's not?